src/slot_machine.c

```diff
@@ -31,18 +31,39 @@
 #define SLOTMACHINE_GFX_TILES 233
 #define MAX_BET 3
 
-#define SYMBOLS_PER_REEL      21
-#define REEL_SYMBOL_HEIGHT    24
-
-// Lucky Flags
-#define LUCKY_BIAS_REPLAY    (1 << 0)
-#define LUCKY_BIAS_CHERRY    (1 << 1)
-#define LUCKY_BIAS_LOTAD     (1 << 2)
-#define LUCKY_BIAS_AZURILL   (1 << 3)
-#define LUCKY_BIAS_POWER     (1 << 4)
-#define LUCKY_BIAS_REELTIME  (1 << 5)
-#define LUCKY_BIAS_MIXED_777 (1 << 6)
-#define LUCKY_BIAS_777       (1 << 7)
+#define TAGS_PER_REEL 21
+#define TAG_HEIGHT    24
```

Comment:
I'm not sure I like `slot_machine`'s usage of "tag" to refer to the symbols on the reels. Does anything in-game (or for a real slot machine) refer to these as "tags"? At a glance, it's more obvious to me what `REEL_SYMBOL_HEIGHT` means as opposed to `TAG_HEIGHT`

Reply:
I mainly picked it because it's shorter than "symbol", which I thought would help shorten function names, vars, comments, etc. But "symbol" does seem a bit more appropriate, so if you'd like, I can switch to this.

Reply:
Yeah it's not that much shorter, and I think it could be confusing both because it's not immediately clear what it refers to and "tag" is used commonly in the repo to refer to tile/palette identifier values.

Reply:
Gotcha, okay I switched to "symbol" nomenclature here.